### PR TITLE
Eric/bug fix

### PR DESCRIPTION
**Pull request by ecorbisierSimplon**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [x] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Checklist

- [x] My PR has a related issue.
- [x] I correctly filled the PR template
- [x] I self reviwed my changes
- [x] I manually check my change

## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_

- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by mohameddiaw — APPROVED** [review]

File: frontend/src/routes/login/+page.server.ts [hunk 1]
```diff
@@ -79,13 +86,11 @@ export const actions = {
 		} catch (error) {
 			console.error('Error during login:', error);
 			return null;
-
 		}
 	}
-	return { success: true };
+	// return { success: true };
```
Comment (Sylvana7) [inline frontend/src/routes/login/+page.server.ts:91]: remove this unused code
```suggestion
```

File: .env/backend.env [hunk 2]
```diff
@@ -1,5 +1,6 @@
 NODE_ENV=development
 PORT=3000
+PASS=PassW0rd!
```
Comment (Sylvana7) [inline .env/backend.env:3]: ⚠️ secrets shouldn't be commited

File: backend/src/users/user.service.ts [hunk 3]
```diff
@@ -14,20 +14,30 @@ export class UserService {
   ) {}
 
   async register(user: CreateUserDto) {
+    const count = await this.countUser();
+    console.log(process.env.PASS + ' // ' + user.password_first);
```
Comment (Sylvana7) [inline backend/src/users/user.service.ts:18]: remove this console.log

File: frontend/src/routes/register/+page.server.ts [hunk 4]
```diff
@@ -5,33 +5,81 @@ import type { PageServerLoad, Actions } from './$types';
...
+			messagesError = { ...messagesError, passwordError: testInput };
 		}
 
-		if (ValidateForm.validateField(sur_name.toString(), 'surname')) {
-			return fail(400, { sur_name, surnameError: 'required' });
+		// testInput = ValidateForm.validatePassword(password_validation.toString());
+		// if (testInput) {
+		// 	codes = 400;
+		// 	messagesError = { ...messagesError, password_validationError: testInput };
+		// }
```
Comment (Sylvana7) [inline frontend/src/routes/register/+page.server.ts:60]: remove unused code

**Review by Sylvana7 — CHANGES_REQUESTED** [review]
try to make smaller and respect the scope of the PR by solving one ticket at a time

File: .env/backend.env (hunk 2 above)
Comment (ecorbisierSimplon) [reply .env/backend.env:3]: Yes, you're right, but in this case it's for testing, not production. The password must be changed for production.

**Review by Sylvana7 — APPROVED** [review]

**Pull request merged**